front/app/components/SeatInfo/BillingInfo.test.tsx

```diff
@@ -0,0 +1,118 @@
+import React from 'react';
+import { render, screen } from 'utils/testUtils/rtl';
+import { IAppConfigurationSettingsCore } from 'api/app_configuration/types';
+
+import BillingInfo from './BillingInfo';
+
+type MockAppConfigurationType = {
+  data: {
+    id: string;
+    attributes: {
+      settings: {
+        core: {
+          maximum_admins_number: IAppConfigurationSettingsCore['maximum_admins_number'];
+          maximum_moderators_number: IAppConfigurationSettingsCore['maximum_moderators_number'];
+          additional_admins_number: IAppConfigurationSettingsCore['additional_admins_number'];
+          additional_moderators_number: IAppConfigurationSettingsCore['additional_moderators_number'];
+        };
+      };
+    };
+  };
+};
+
+let mockAppConfiguration: MockAppConfigurationType = {
+  data: {
+    id: '1',
+    attributes: {
+      settings: {
+        core: {
+          maximum_admins_number: 6,
+          maximum_moderators_number: 9,
+          additional_admins_number: 0,
+          additional_moderators_number: 0,
+        },
+      },
+    },
+  },
+};
+
+jest.mock('api/app_configuration/useAppConfiguration', () => () => {
+  return { data: mockAppConfiguration };
+});
+
+let mockUserSeatsData = {
+  data: {
+    attributes: {
+      admins_number: 2,
+      project_moderators_number: 5,
+    },
+  },
+};
+
+jest.mock('api/seats/useSeats', () => () => {
+  return {
+    data: mockUserSeatsData,
+  };
+});
+
+describe('BillingInfo', () => {
+  beforeEach(() => {
+    mockUserSeatsData.data.attributes.admins_number = 2;
+    mockUserSeatsData.data.attributes.project_moderators_number = 5;
+
+    mockAppConfiguration.data.attributes.settings.core.maximum_admins_number = 6;
+    mockAppConfiguration.data.attributes.settings.core.maximum_moderators_number = 9;
+    mockAppConfiguration.data.attributes.settings.core.additional_admins_number = 0;
+    mockAppConfiguration.data.attributes.settings.core.additional_moderators_number = 0;
+  });
+
+  it('shows correct numbers of seat usage for admins', () => {
+    render(<BillingInfo seatType="admin" />);
+    expect(screen.getByText('Admin seats')).toBeInTheDocument();
+
+    // Remaining seats
+    expect(screen.getByText('Remaining seats')).toBeInTheDocument();
+    expect(screen.getByText('4')).toBeInTheDocument();
+
+    // Used seats
+    expect(screen.getByText('Used seats')).toBeInTheDocument();
+    expect(screen.getByText('2')).toBeInTheDocument();
+
+    // Total seats
+    expect(screen.getByText('Total seats')).toBeInTheDocument();
+    expect(screen.getByText('6')).toBeInTheDocument();
+  });
```

Comment:
Can we also still add tests for correct display of "within plan" and "additional"? Or prefer to do that after the copy review? Create a ticket in that case.

<img width="172" alt="Screenshot 2023-04-06 at 11 09 59" src="https://user-images.githubusercontent.com/16427929/230330769-3ab22621-2a9c-4072-8818-3891e99293f8.png">

Reply:
(Same for collaborators)